src/tir/ir/sparse.cc

```diff
@@ -234,7 +234,6 @@ SpIterVar::SpIterVar(Var var, PrimExpr max_extent, SpIterKind kind, bool is_redu
   ObjectPtr<SpIterVarNode> node = make_object<SpIterVarNode>();
 
   arith::Analyzer ana;
-  CHECK(ana.CanProveEqual(axis->length, max_extent));
```

Comment:
Because for sparse-fixed axes, what we need to prove is "`max_extent` <= `axis->length`". But in symbolic stage the lack of numeric information makes it impossible to prove the inequality.

Reply:
sounds reasonable.